include.js

```diff
@@ -493,6 +493,13 @@ let cards = [
     author: 'Jason',
     githubLink: 'https://github.com/jsonsinger'
   },
+  {
+    artName: "Bubble Art",
+    pageLink: './Art/Comder/index.html',
+    imageLink: './Art/Comder/Animation.png',
```

Comment:
## Feedback - wrong file path
- the folder is not Comder but Comder101: `./Art/Comder101/`

## Context
```js
  pageLink: './Art/Comder/index.html',
  imageLink: './Art/Comder/Animation.png',
```

Reply:
Oh, Thanks alot totally missed it ; will make the requested changes and create new pr.